### PR TITLE
updates to electron-packager and node-notifier for CVEs

### PR DESCRIPTION
**Pull request by birarda**
- updated two server-console dependencies for security updates for module dependencies

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7135/app_12385_PR.apk

**Comment by birarda** [comment]
#### Test Plan

This PR updates the packaging and notifier modules for the electron based Sandbox.

1. Run the Sandbox from this PR. Confirm that it launches as expected.
2. On launch, though there is actually no update available, a system notification should appear that says "TEST: An update is available!". I have manually added that notification to each run of the Sandbox for testing. It is to be removed after QA approval and before merging this PR.
3. Launch Interface from this PR, or from master. Connect to the Sandbox running from this PR. Ensure that you connect to the domain and all assignment clients as expected.
4. Ensure that you can reach the domain settings from the Sandbox tray menu.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7136/app_12385_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7136/HighFidelity-Beta-PR12385-761232fcdac0ac7985a58dcf6cc5fde497ca08d6.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7136/HighFidelity-Beta-PR12385-761232fcdac0ac7985a58dcf6cc5fde497ca08d6.exe

**Comment by sbennett77** [comment]
Currently Testing.

**Comment by sbennett77** [comment]
Pass

@conklin94122

File: server-console/src/main.js [hunk 1]
```diff
@@ -860,6 +860,13 @@ function onContentLoaded() {
     // Disable splash window for now.
     // maybeShowSplash();
 
+    notifier.notify({
```
Comment (ZappoMan) [inline server-console/src/main.js]: Approved to merge once this is removed.

**Comment by birarda** [comment]
@ZappoMan removed the test notification

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7179/app_12385_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7179/HighFidelity-Beta-PR12385-f95893f070a0552b1fd30ec7b5de1fea3018d8dc.dmg

**Comment by birarda** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7197/app_12385_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7197/HighFidelity-Beta-PR12385-f95893f070a0552b1fd30ec7b5de1fea3018d8dc.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7197/HighFidelity-Beta-PR12385-f95893f070a0552b1fd30ec7b5de1fea3018d8dc.dmg

**Pull request merged**